tests/unit/core/reducers/test_addons.js

```diff
@@ -49,37 +58,48 @@ describe('addon reducer', () => {
       slug: 'installable',
       icon_url: 'http://foo.com/img.png',
     };
+
     expect(
-      addons(undefined, { payload: { entities: { addons: { installable: addon } } } })
+      addons(undefined, loadEntities(createFetchAddonResult(addon).entities))
     ).toEqual({
       installable: {
         ...addon,
-        iconUrl: 'http://foo.com/img.png',
+        iconUrl: addon.icon_url,
       },
     });
   });
 
   it('flattens theme data', () => {
-    const type = ADDON_TYPE_THEME;
-    const state = addons(originalState, {
-      payload: {
-        entities: {
-          addons: {
-            baz: { slug: 'baz', id: 42, theme_data: { theme_thing: 'some-data' }, type },
-          },
-        },
-      },
+    const theme = {
```

Comment:
this should extend from `fakeAddon` to future proof the properties (in case new required properties are introduced).